chapter2/tests/t2/chapter2-t2-gen-kustomization.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+#
+# Task 2 - Generate kustomization.yaml
+set -eo pipefail
+
+
+# Error handling function.
+err() {
+  echo "[$(date +'%Y-%m-%dT%H:%M:%S%z')]: $*" >&2
+}
+
+
+# Checks (perms, args, dir-exist)
+if ! [[ -r $(pwd) ]]; then
+  err "Directory is not readable."
+  exit 255
+fi
+
+if [[ "$#" -lt 1 ]]; then
+  err "At least 1 or more args are expected"
+  exit 1
+fi
+
+if [[ ! -d "repos" ]]; then
+  mkdir repos
+fi
+
+
+# ssh-keys generation
+for arg in $@; do
+  ssh-keygen -t ed25519 -N "" -C "$arg" -f ./repos/$arg-deploy-key.pem;
```

Comment:
1. Let's not ask user to overwite existing key and let's always create a new key (i.e. you should tell this command to always create new file)
2. Let's hide this output `ssh-keygen` output
![image](https://user-images.githubusercontent.com/26448672/229416005-964501fb-7798-421e-893d-448172adada2.png)

Reply:
@kseniyashaydurova 
Everything is done, please check.